vendor/bat-native-ads/src/bat/ads/internal/ads/new_tab_page_ads/new_tab_page_ad_unittest.cc

```diff
@@ -103,17 +101,14 @@ class BatAdsNewTabPageAdTest : public NewTabPageAdObserver,
 };
 
 TEST_F(BatAdsNewTabPageAdTest, FireViewedEvent) {
-  // Arrange
-  new_tab_page_ads::frequency_capping::ForcePermissionRules();
-
   const CreativeNewTabPageAdInfo& creative_ad = BuildAndSaveCreativeAd();
 
   // Act
   new_tab_page_ad_->FireEvent(kUuid, creative_ad.creative_instance_id,
                               mojom::NewTabPageAdEventType::kViewed);
 
   // Assert
-  EXPECT_TRUE(did_serve_ad_);
```

Comment:
See previous comment concerning `This logic is still required if ads are disabled`

Reply:
Moved these tests to a separate file: [new_tab_page_ad_if_ads_disabled_unittest.cc](https://github.com/brave/brave-core/pull/11945/files#diff-6982bf23f496560ee920b6c2550eae466aaba9a5042982abd56169e5e9b53272)

These tests are running with Brave Ads preference disabled.